src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -357,20 +361,34 @@ public void onItemCheckedStateChanged(ActionMode mode, int position, long id, bo
                     mAdapter.removeSelection(position);
                 }
 
-                // TODO maybe change: only recreate menu if count changes
-                if (menu != null) {
-                    menu.clear();
-                    if (checkedCount == 1) {
-                        createContextMenu(menu);
-                    } else {
-                        // download, move, copy, delete
-                        getActivity().getMenuInflater().inflate(R.menu.multiple_file_actions_menu, menu);
+                OCFile targetFile = null;
+                if (checkedCount > 0 && checkedCount <= MIN_FILES_FOR_MULTISELECT) {
```

Comment:
if (checkedCount == 1) has been removed so how will the multi select behave if you only selected one item? @tobiasKaminsky's implementation showed a different set of actions depending on the number of selected items (1 or more than 1